src/main/java/com/julienvey/trello/impl/TrelloImpl.java

```diff
@@ -510,6 +438,18 @@ public void addLabelsToCard(String idCard, String[] labels) {
         }
     }
 
+    @Override
+    public CustomFieldItem updateCustomFieldItem(String idCard, CustomFieldItem customFieldItem) {
+        Objects.requireNonNull(idCard);
+        Objects.requireNonNull(customFieldItem);
+
+        customFieldItem = put(createUrlWithNoArgs(UPDATE_CUSTOM_FIELD_ITEM), customFieldItem, CustomFieldItem.class, idCard,
+                customFieldItem.getIdCustomField());
+        customFieldItem.setCardId(idCard);
```

Comment:
similarly here - since this method needs to return an object, according to its signature, then we should create one and return it.
this method currently mixes two paradigms - returning a new object and changing value on the parameter . it should be one or the other.